build.gradle

```diff
@@ -131,8 +131,14 @@ project(':iceberg-relocate-external-dependencies') {
       include 'NOTICE'
     }
 
+    dependencies {
+      exclude(dependency('com.github.stephenc.findbugs:findbugs-annotations'))
```

Comment:
Can you move these up into the section starting on line 115 so they are all in the same place?

Reply:
I'm not sure this can be moved, I think this exclude(dependency(..)) thing is part of the shadowJar plugin, so it needs to happen here rather than in the module dependencies section

Reply:
Yes, you're right, sorry, I missed that.